projects/components/select/src/select.component.ts

```diff
@@ -89,6 +89,13 @@ export class PsSelectComponent<T = unknown> extends _PsSelectMixinBase
 
   @ViewChild(MatSelect, { static: true }) public set setMatSelect(select: MatSelect) {
     this._matSelect = select;
+
+    // MatSelect doesn't trigger stateChanges on close which causes problems, so we patch it here.
+    const close = select.close;
```

Comment:
please add a test for the bug this fixes

Reply:
I enabled the existing test for that.